### PR TITLE
Fastkml

### PR DESCRIPTION
**Pull request by dcunn**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fastkml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - pygeoif
+    - python
+    - python-dateutil
+
+test:
+  imports:
+    - fastkml
+  requires:
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/fastkml/meta.yaml]: as you are not running the tests, could you please remove the 
```
  requires:
    - pytest
```

File: recipes/fastkml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+about:
+  home: "https://github.com/cleder/fastkml"
+  license: "GNU Library or Lesser General Public (LGPL)"
```
Comment (marcelotrevisani) [inline recipes/fastkml/meta.yaml]: I think it should be
```suggestion
  license: "LGPL-2.1"
```

File: recipes/fastkml/meta.yaml (hunk 1 above)
Comment (dcunn) [reply recipes/fastkml/meta.yaml]: @marcelotrevisani removing test caused build to fail

Comment (chrisburr) [reply recipes/fastkml/meta.yaml]: The failure is because `setuptools` is missing from the `run` dependencies

Comment (dcunn) [reply recipes/fastkml/meta.yaml]: Thank you @chrisburr

**Pull request merged**